addon/components/simple-typeahead/template.hbs

```diff
@@ -1,16 +1,25 @@
 <div class="input">
-  <input
+  {{input
```

Comment:
what's the benefit of using the `{{input ...}}` helper over the native html `<input>` element?  I remember changing the implementation from the ember helper to the native element b/c I didn't like something about the helper's implementation.  Can't remember what specifically, but had to do w/ the 2-way bindings (which Ember suggests you don't use, but continues to implement in cases like this one...)

Is there functionality specifically related to `{{input}}` that you don't get from `<input>` that prompted this change, or did you change just to be more emberish?